sound/soc/sof/control.c

```diff
@@ -15,6 +15,9 @@
 #include "sof-priv.h"
 #include "sof-audio.h"
 
+/* Interface version implemented for controls IPC */
+#define SOF_CONTROL_ABI_VERSION SOF_ABI_VER(3, 18, 0)
```

Comment:
Where is the authoritative list of new features added to each MINOR increment? Is it https://github.com/orgs/thesofproject/projects/2? Either way, add a link to it somewhere in the source code?